ddtrace/tracer/writer_test.go

```diff
@@ -275,6 +275,22 @@ func TestLogWriterOverflow(t *testing.T) {
 		assert.Equal(io.EOF, err)
 	})
 }
+func TestJsonEncodeSpanNewLines(t *testing.T) {
+	assert := assert.New(t)
+	s := newSpan("name", "srv", "res", 2, 1, 3)
+	s.Start = 12
+	s.Meta["query"] = "Select * from \n Where value"
+	s.Meta["query\n"] = "Select * from \n Where\nvalue"
+
+	h := &logTraceWriter{}
+	h.resetBuffer()
```

Comment:
```suggestion
	var h logTraceWriter
```
Why do we need to reset something that is new?